### PR TITLE
Skip `-race` flag on Windows

### PR DESCRIPTION
**Pull request by w84thesun**
Refs #388.

### CODE REVIEW

**Review by seeforschauer — APPROVED** [review]

**Review by AlekSi — CHANGES_REQUESTED** [review]
That's not the place where we have `-race`

File: Taskfile.yml [hunk 1]
```diff
@@ -168,7 +168,7 @@ tasks:
     desc: "Build bin/ferretdb-testcover"
     deps: [gen-version]
     cmds:
-      - go test -c -o=bin/ferretdb-testcover -trimpath -tags=testcover,tigris -race -coverpkg=./... ./cmd/ferretdb
+      - go test -c -o=bin/ferretdb-testcover -trimpath -tags=testcover,tigris {{if not (eq OS "windows")}} -race {{end}} -coverpkg=./... ./cmd/ferretdb
```
Comment (AlekSi) [inline Taskfile.yml:171]: Can we use `ne` (not equal) operator?

Comment (w84thesun) [reply Taskfile.yml]: Yes, changed to `ne`.

**Comment by w84thesun** [comment]
> That's not the only place where we have `-race`

Added to all other places.

**Review by AlekSi — APPROVED** [review]

**Review by seeforschauer — APPROVED** [review]

**Review by noisersup — APPROVED** [review]

**Pull request merged**